pages/documentation/download.html

```diff
@@ -24,6 +24,12 @@ <h2 class="usa-heading">Code</h2>
 
 <!--<span class="link-download-subtext">Version {{ site.uswds_version }}</span>-->
 
+<!-- 
+  This note also exists on the Documentation page, which is populated by the USWDS README. 
+  If any changes are made, make sure to update there as well. 
+-->
```

Comment:
Following [liquid docs](https://shopify.github.io/liquid/tags/template/), we can use a comment that won't render. 

```suggestion
{% comment %}
  This note also exists on the Documentation page, which is populated by the USWDS README.
  If any changes are made, make sure to update there as well.
{% endcomment %}
```

![image](https://github.com/uswds/uswds-site/assets/3385219/3740f02a-3a4e-436c-9d99-626b979cd583)